app/views/transport_requisitions/show.html.erb

```diff
@@ -0,0 +1,252 @@
+            Certified by:
+          </div>
+          <div class="col-md-8 col-sm-8 col-xs-8">
+            <% if @transport_requisition.certified_by.present? %>
+              <%= User.find(@transport_requisition.certified_by).name %>
+            <% else %>
+              { pending }
+            <% end %>
+          </div>
+        </div>
+        <div class="form-group col-md-12 col-sm-12 col-xs-12">
+          <div class="col-md-4 col-sm-4 col-xs-4 text-right">
+            Created date:
+          </div>
+          <div class="col-md-8 col-sm-8 col-xs-8">
+          <% if @transport_requisition.created_at.present? %>
+            <%= @transport_requisition.created_at.to_formatted_s(:long_ordinal)  %>
+          <% else %>
+            { pending }
+          <% end %>
+          </div>
+        </div>
+        <div class="form-group col-md-12 col-sm-12 col-xs-12">
+          <div class="col-md-4 col-sm-4 col-xs-4 text-right">
+            Certified date:
+          </div>
+          <div class="col-md-8 col-sm-8 col-xs-8">
+          <% if @transport_requisition.certified_date.present? %>
+            <%= @transport_requisition.certified_date.to_formatted_s(:long_ordinal)  %>
+          <% else %>
+            { pending }
+          <% end %>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="ibox col-sm-12" >
+    <div class="ibox-content col-sm-12" style="height:1100px">
+      <div class="panel blank-panel">
+        <div class="panel-heading">
+          <div class="panel-options">
+            <ul class="nav nav-tabs">
+              <li class="active">
+                <a href="#tab-1" data-toggle="tab">Commodity</a>
+              </li>
+              <li class="">
+                <a href="#tab-2" data-toggle="tab">Destinations</a>
+              </li>
+              <li class="">
+                <a href="#tab-3" data-toggle="tab">Transporters</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+
+        <div class="panel-body col-sm-12" style="height:1100px">
+
+          <div class="tab-content">
+            <div class="tab-pane active col-sm-12 row" id="tab-1">
+              <%if(!@transport_requisition.present?)%>
+                <h2>
+                    No requisitions are found under this transport requisition<br>
+                </h2>
+              <%else%>
+                <table class="table table-hover cats-datatable">
+                  <thead>
+                    <tr>
+                      <th>Requisition#</th>
+                      <th>Commodity</th>
+                      <th>Quantity(mt)</th>
+                      <th>Warehouse</th>
+                      <th>Region</th>
+                      <th>Zone</th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    <% @transport_requisition.transport_requisition_items.pluck(:requisition_id).uniq.each do |req_id| %>
```

Comment:
You are iterating over a list of requisition items in a TR and hitting the DB to get commodity, location and FDP information for each row. This will make the page feel like as if it's made of bricks - try including related records in the controller logic than invoking the model from the view. If you think about it this is a clear violation of the MVC pattern - accessing the model directly from the view -> not so good approach!